db/migrate/20220707171901_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, foreign_key: true
```

Comment:
- [ ] It should be the author, not the user. Kindly change it.

You can do it like in the picture.
![image](https://user-images.githubusercontent.com/38635158/177837184-ec62ca8a-3a06-4c56-b300-fa2820ebe958.png)